File: AppServer/DBMapping/src/DBClasses/Place.java
package DBClasses;

public class Place {
    public int Id;
    public String Cuisine;
    public double Height;
    public int PlaceId;
    public String PlaceType;
    public double PriceRange;
    public double Width;
    public Place(){}
    public Place(int id, String cuisine,double height, int placeId, String placeType, double priceRange, double width )
    {
        Id=id;
        Cuisine=cuisine;
        Height = height;
        PlaceId = placeId;
        PlaceType = placeType;
        PriceRange = priceRange;
        Width = width;
    }

}

File: AppServerC/AppServerC/DAL/Place.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace AppServerC
{
    public partial class Place
    {
        public Place()
        {
            Favourites = new HashSet<Favourite>();
            Histories = new HashSet<History>();
            PlaceSchemes = new HashSet<PlaceScheme>();
        }

        public int Id { get; set; }
        public int? PlaceId { get; set; }
        public double? Height { get; set; }
        public double? Width { get; set; }
        public int? PriceRange { get; set; }
        public string PlaceType { get; set; }
        public string Cuisine { get; set; }

        public virtual ChainName PlaceNavigation { get; set; }
        public virtual ICollection<Favourite> Favourites { get; set; }
        public virtual ICollection<History> Histories { get; set; }
        public virtual ICollection<PlaceScheme> PlaceSchemes { get; set; }
    }
}
